src/commonComponents/button/Button.js

```diff
@@ -0,0 +1,51 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Constants
+import { BUTTON_TYPES } from './constants/button.general';
+
+//Helpers
+import _noop from 'lodash/noop';
+import _values from 'lodash/values';
+import cx from 'classnames';
+
+//CSS
+import './button.css';
+
+function Button(props) {
+  const { label, onClick, className, type, id, ...rest } = props;
+  const typeClassName = cx({
+    [BUTTON_TYPES.PRIMARY]: type === BUTTON_TYPES.PRIMARY,
+    [BUTTON_TYPES.SECONDARY]: type === BUTTON_TYPES.SECONDARY,
+  });
+  function handleClick(e) {
+    return onClick(id);
```

Comment:
since button is a common component it will be used for all the usecases, here since you are not forwarding event there could be a usecase where you need this event, to solve this just pass onClick handler to button component and don't do this in common component, you want to do this where it will not affect all the flows